<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWindow</class>
 <widget class="QWidget" name="PlotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainHorizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Plot" name="plot" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid black;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="plotParametersLayout">
       <item>
        <spacer name="plotParametersVerticalSpacer1">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lineWidthLabel">
         <property name="text">
          <string>Товщина лінії:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="lineWidthSpinBox">
         <property name="singleStep">
          <double>0.250000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="plotParametersVerticalSpacer2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lineColorLabel">
         <property name="text">
          <string>Колір:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lineRedLayout">
         <item>
          <widget class="QLabel" name="lineRedLabel">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="lineRedSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lineGreenLayout">
         <item>
          <widget class="QLabel" name="lineGreenLabel">
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="lineGreenSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lineBlueLayout">
         <item>
          <widget class="QLabel" name="lineBlueLabel">
           <property name="text">
            <string>B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="lineBlueSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lineAlphaLayout">
         <item>
          <widget class="QLabel" name="lineAlphaLabel">
           <property name="text">
            <string>A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="lineAlphaSlider">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="plotParametersVerticalSpacer3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lineStyleLabel">
         <property name="text">
          <string>Стиль лінії:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="lineStyleSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="plotParametersVerticalSpacer4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QWidget</extends>
   <header>Plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineWidthSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlotWindow</receiver>
   <slot>changePlotStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineRedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PlotWindow</receiver>
   <slot>changePlotStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineGreenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PlotWindow</receiver>
   <slot>changePlotStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineBlueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PlotWindow</receiver>
   <slot>changePlotStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineStyleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PlotWindow</receiver>
   <slot>changePlotStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changePlotStyle()</slot>
 </slots>
</ui>
